amundsen_application/static/js/ducks/lastIndexed/tests/index.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { testSaga } from 'redux-saga-test-plan';
+
+import * as API from '../api/v0';
+
+import reducer, {
+  getLastIndexed,
+  getLastIndexedFailure,
+  getLastIndexedSuccess,
+  initialState,
+  LastIndexedReducerState,
+} from '../reducer';
+
+import { getLastIndexedWatcher, getLastIndexedWorker } from '../sagas';
+import { GetLastIndexed } from '../types';
+
+describe('lastIndexed ducks', () => {
+  let testEpoch: number;
+
+  beforeAll(() => {
+    testEpoch = 1545925769;
+  });
+
+  describe('actions', () => {
+    it('getLastIndexed - returns the action to get the last indexed date', () => {
+      const action = getLastIndexed();
+      expect(action.type).toBe(GetLastIndexed.REQUEST);
+    });
+
+    it('getLastIndexedFailure - returns the action to process failure', () => {
+      const action = getLastIndexedFailure();
+      expect(action.type).toBe(GetLastIndexed.FAILURE);
+    });
+
+    it('getLastIndexedSuccess - returns the action to process success', () => {
+      const action = getLastIndexedSuccess(testEpoch);
+      const { payload } = action;
+      expect(action.type).toBe(GetLastIndexed.SUCCESS);
+      expect(payload.lastIndexedEpoch).toBe(testEpoch);
+    });
+  });
+
+  describe('reducer', () => {
+    let testState: LastIndexedReducerState;
+    beforeAll(() => {
+      testState = initialState;
+    });
+
+    it('should return the existing state if action is not handled', () => {
+      expect(reducer(testState, { type: 'INVALID.ACTION' })).toEqual(testState);
+    });
+
```

Comment:
We are missing a test for the REQUEST, check here for an example: https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/ducks/dashboard/tests/reducer.spec.ts#L24